File: examples/TeamAddMemberExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
  # Configure HTTP basic authorization: api_key
  config.username = "YOUR_API_KEY"

  # or, configure Bearer (JWT) authorization: oauth2
  # config.access_token = "YOUR_ACCESS_TOKEN"
end

team_api = Dropbox::Sign::TeamApi.new

data = Dropbox::Sign::TeamAddMemberRequest.new
data.email_address = "george@example.com"

begin
  result = team_api.team_add_member(data)
  p result
rescue Dropbox::Sign::ApiError => e
  puts "Exception when calling Dropbox Sign API: #{e}"
end

File: examples/TeamAddMemberExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TeamAddMemberExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        var teamAddMemberRequest = new TeamAddMemberRequest(
            emailAddress: "george@example.com"
        );

        try
        {
            var response = new TeamApi(config).TeamAddMember(
                teamAddMemberRequest: teamAddMemberRequest,
                teamId: "4fea99bfcf2b26bfccf6cea3e127fb8bb74d8d9c"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling TeamApi#TeamAddMember: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
